CMakeLists.txt

```diff
@@ -125,6 +125,7 @@ if (${USE_PODIO})
     find_package(podio REQUIRED)
     add_compile_definitions(HAVE_PODIO)
     include_directories(SYSTEM ${podio_INCLUDE_DIR})
+    link_libraries(podio::podio podio::podioRootIO)
```

Comment:
I have been using a  following fix locally
```suggestion
    target_link_libraries(jana2_shared_lib podio::podio podio::podioRootIO)
```

Reply:
I had that at first too, in src/libraries/JANA/CMakeLists.txt (with a USE_PODIO guard), but then I decided to just follow the global approach already in use and put in the note about someone (else) needing to revamp the whole CMakeLists setup...

Reply:
Either is fine by me!